Tests/NIOSSLTests/ClientSNITests.swift

```diff
@@ -79,17 +79,70 @@ class ClientSNITests: XCTestCase {
 
     func testSNIIsRejectedForIPv4Addresses() throws {
         let context = try configuredSSLContext()
-        
-        XCTAssertThrowsError(try NIOSSLClientHandler(context: context, serverHostname: "192.168.0.1")){ error in
+
+        let testString = "192.168.0.1"
+        XCTAssertThrowsError(try NIOSSLClientTLSProvider<ClientBootstrap>(context: context, serverHostname: testString))
```

Comment:
Is there a reason this doesn't assert the specific error type? This question applies throughout this file, as this pattern is very consistent.

Reply:
whoops, fixed.